java/MinePump/spec1-5_product1.yml

```diff
@@ -0,0 +1,6 @@
+format_version: "0.1"
+input_files: spec1-5_product1_false-assert/src/main/java/
+properties:
+  - property_file: ../properties/assert.prp
```

Comment:
Just a question: is it also possible to inline properties here in future? (e.g. to avoid duplicating source code from which different benchmarks are derived by running from different entry points)

Reply:
No: verifiers need to be given the property, so the property should be present as a separate file, and not inlined in the YAML file. But of course one can have one property file per entry point. There is also no need to duplicate source code: several YAML files can refer to the same source code.

Reply:
So, property files need not necessarily live in `properties`, but could be called `java/bigbenchmarks/mybigbenchmark_prop1.prp` alongside a `java/bigbenchmarks/mybigbenchmark_prop1.yml`?

Reply:
In principle yes.
But if we keep .prp files everywhere, then we will get lots of duplicates.
If there is a .prp file that is very specific to a particular program, then it could well live next to the YML file that is referring to it.